types/twine-sugarcube/state.d.ts

```diff
@@ -164,6 +164,30 @@ export interface StateAPI {
          */
         delete(key: string): void;
 
+        /**
+         * Returns an array of the story metadata store's key/value pairs as [key, value] arrays.
+         * @since 2.36.0
+         * @example
+         * // Get the metadata store's key/value pairs.
+         * var metadata = State.metadata.entries();
+         *
+         * // Iterate over the pairs with a `for` loop.
+         * for (var i = 0; i < metadata.length; ++i) {
+         *      var key   = metadata[i][0];
+         *      var value = metadata[i][1];
+         * // do something
+         * }
+         *
+         * @example
+         * // Iterate over the pairs with `<Array>.forEach()`.
+         * State.metadata.entries().forEach(function (pair) {
+         * 	var key   = pair[0];
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/twine-sugarcube/state.d.ts;
  Line: 184;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.